src/Servers/Kestrel/Core/src/KestrelServerOptions.cs

```diff
@@ -91,14 +93,6 @@ public class KestrelServerOptions
         /// </remarks>
         public bool DisableStringReuse { get; set; }
 
-        /// <summary>
-        /// Controls whether to return the "Alt-Svc" header from an HTTP/2 or lower response for HTTP/3.
-        /// </summary>
-        /// <remarks>
-        /// Defaults to false.
-        /// </remarks>
-        public bool EnableAltSvc { get; set; }
```

Comment:
This is a breaking change, right? It's weird that the build isn't complaining that the PublicAPI files haven't changed.

Can we just leave this around and keep it as a default? And make the ListenOptions version nullable? Or we can just make an announcement and take the breaking change. I'd be surprised if it actually broke anyone, but this public property existed in 5.0.

Reply:
:+1: standard Obsolete and announce protocol.

Reply:
It was added in .NET 5 for HTTP/3. And HTTP/3 in .NET 5 basically supported Hello World and nothing else. IMO we can skip the formalities are remove it in .NET 6.

Reply:
Blah, I'll just go the normal route and obsolete. Less work that discussion over being different 😋